src/Features/CSharp/Portable/QualifyMemberAccess/CSharpQualifyMemberAccessDiagnosticAnalyzer.cs

```diff
@@ -13,5 +13,9 @@ protected override string GetLanguageName()
 
         protected override bool IsAlreadyQualifiedMemberAccess(SyntaxNode node)
             => node.IsKind(SyntaxKind.ThisExpression);
+
+        // If the member is already qualified with `base.`, it cannot be further qualified.
+        protected override bool CanMemberAccessBeQualified(SyntaxNode node)
+            => !node.IsKind(SyntaxKind.BaseExpression);
```

Comment:
@CyrusNajmabadi This was answered in this comment: https://github.com/dotnet/roslyn/pull/19812#discussion_r118935090